sdk/rotation/rotation_job.go

```diff
@@ -26,41 +26,46 @@ type RotationJob struct {
 	// For requests, this will always be blank.
 	RotationID string `sentinel:""`
 	Path       string
+	MountPoint string
+	MountType  string
 	Name       string
 }
 
 type RotationJobConfigureRequest struct {
 	Name             string
 	MountPoint       string
+	MountType        string
```

Comment:
Is it necessary to pass MountPoint and MountType to Vault core from the plugin? Can they be inferred with the RotationManager's router.MatchingMountEntry? It seems redundant since Vault core knows all that info already.

Reply:
Good call! We should be able to obtain this using the MatchingMountEntry

Reply:
Removed the need for extra mount info like `MountPoint` to be passed from the plugin. We can get that info from other avenues in Vault Core